NanoVNASaver/RFTools.py

```diff
@@ -170,3 +170,168 @@ def corr_att_data(data: List[Datapoint], att: float) -> List[Datapoint]:
         corrected = dp.z * att
         ndata.append(Datapoint(dp.freq, corrected.real, corrected.imag))
     return ndata
+
+def match_calc_string_value_with_prefix(val,unit):
```

Comment:
Is this something that the Value class of SITools can't do?
like 
print(Value("10p", unit="F"))
print(float(Value("1m") * 1000))
print(Value(10e-9))

If there is something missing, it would be better in that class, instead of this if/else mess as unrelated function